### PR TITLE
Added Buttons

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Added Buttons -
   a. Contained Buttons (Filled)
   b. Outline Buttons
   c. Icon Buttons
   d. Floating Action Buttons
   e. Link
   f. Button with Icons
   g. Different sizes buttons - small, medium, large
   
Buttons

### CODE REVIEW

File: styles/components/buttons.css [hunk 1]
```diff
@@ -1,11 +1,146 @@
...
+  cursor: pointer;
 }
 
-.button:hover {
-  background-color: rgb(0, 0, 0, 0.2);
+.button:disabled,
+.button[disabled] {
+  border: 1px solid #999999;
+  background-color: #cccccc;
+  color: #666666;
```
Comment (rutvikpumak) [inline styles/components/buttons.css:16]: use color variable

**Review by Prabhav-Sharma — APPROVED** [review]

File: styles/components/buttons.css [hunk 2]
```diff
@@ -1,11 +1,146 @@
-.button {
+.button,
+a.button {
```
Comment (rutvikpumak) [inline styles/components/buttons.css]: dont play with direct element, play by giving class name

File: styles/components/avatars.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .avatar {
   border-radius: var(--border-radius-circle);
   display: inline-block;
-  margin: 0 0.2rem;
+  margin: 0.1rem;
```
Comment (rutvikpumak) [inline styles/components/avatars.css:4]: why avatars change in button component

File: styles/components/buttons.css (hunk 2 above)
Comment (SukanyaSen530) [reply styles/components/buttons.css]: `button` is the class name

Comment (rutvikpumak) [reply styles/components/buttons.css]: here with a.button you are picking a element directly

File: styles/components/avatars.css (hunk 3 above)
Comment (SukanyaSen530) [reply styles/components/avatars.css:4]: Yeah, will keep it in mind from next PR.

File: styles/components/buttons.css (hunk 2 above)
Comment (rutvikpumak) [reply styles/components/buttons.css]: reviewer might confuse between .button class and button element , better you change class name to something like btn

File: styles/components/buttons.css (hunk 1 above)
Comment (SukanyaSen530) [reply styles/components/buttons.css:16]: Have used color variables for all the common used colors. These will not be used anywhere else.

File: styles/components/buttons.css (hunk 2 above)
Comment (SukanyaSen530) [reply styles/components/buttons.css]: fixed - 93e84e4ca91886737163a6787fe428331992a746

**Review by cjinc121 — APPROVED** [review]
you can go ahead with merging

**Comment by rutvikpumak** [comment]
Looks good now

**Review by tewarimohit — COMMENTED** [review]
Looks fine. Good to merge..

**Pull request merged**